.env

```diff
@@ -7,13 +7,13 @@ RUST_LOG_FORMAT=full
 L1_BLOCK_TIME_SEC=3
 
 # Internal port inside container
-ESPRESSO_SEQUENCER_URL=http://sequencer0:50000
+ESPRESSO_SEQUENCER_URL=http://sequencer0:24000/v0/
 
 # Example rollup demo variables
 ESPRESSO_DEMO_ROLLUP_PORT=8084
 ESPRESSO_DEMO_L1_HTTP_PROVIDER=http://demo-l1-network:8545
 ESPRESSO_DEMO_L1_WS_PROVIDER=ws://demo-l1-network:8546
-ESPRESSO_DEMO_HOTSHOT_ADDRESS=0x0116686e2291dbd5e317f47fadbfb43b599786ef
+ESPRESSO_DEMO_LIGHT_CLIENT_ADDRESS=0x0c8e79f3534b00d9a3d4a856b665bf4ebc22f2ba
```

Comment:
how is this determined?

Reply:
There is a docker container to deploy contracts: https://github.com/EspressoSystems/espresso-sequencer/blob/main/docker-compose.yaml#L32 to local network. The address doesnt change so I added it to `.env`